lib/modules/neverEndingReddit.js

```diff
@@ -167,7 +167,12 @@ function togglePause(pause, source) {
 }
 
 function initiateReturnToPrevPage() {
+	const navigation = window.performance.navigation;
+	// Only restore state if user returned here
+	const returned = ![navigation.TYPE_NAVIGATE, navigation.TYPE_RELOAD].includes(navigation.type);
```

Comment:
Why not `performance.navigation.type === performance.navigation.TYPE_BACK_FORWARD` ?